DependencyInjection/Configuration.php

```diff
@@ -375,10 +375,8 @@ class Configuration implements ConfigurationInterface
 {
     /**
      * Generates the configuration tree builder.
-     *
-     * @return \Symfony\Component\Config\Definition\Builder\TreeBuilder The tree builder
      */
-    public function getConfigTreeBuilder()
+    public function getConfigTreeBuilder(): TreeBuilder
```

Comment:
Technically this is a BC break for anyone that extends Configuration, but let's be pragmatic: nobody does that.

Reply:
Those configuration classes should be `@internal` anyway